test/models/test_cice.py

```diff
@@ -5,32 +5,41 @@
 import f90nml
 
 import payu
+from payu.branch import clone
+import git
+
 
 from test.common import cd
-from test.common import tmpdir, ctrldir, labdir, expt_workdir, ctrldir_basename
+from test.common import tmpdir, ctrldir, labdir, expt_workdir
+from test.common import expt_archive_dir, ctrldir_basename
 from test.common import write_config, write_metadata
-from test.common import make_inputs, make_exe
+from test.common import make_exe
 
 verbose = True
 
+DEFAULT_YEAR_INIT = 101  # arbitrary value for tests
+DEFAULT_DT = 3600  # 1 hour
 DEFAULT_CICE_NML = {
     "setup_nml": {
         "history_dir": "./HISTORY/",
         "restart_dir": "./RESTART/",
-        "year_init": 9999,
-        "days_per_year": 360,
+        "year_init": DEFAULT_YEAR_INIT,
+        "days_per_year": 365,
         "ice_ic": "default",
-        "restart": ".false.",
+        "restart": False,
```

Comment:
Oh yeah, this was something annoying with `f90nml` which was causing problems with the test namelists:
```
>>> import f90nml
>>> mynml = {"section": {"falsevar1":".false.", "falsevar2":False}}
>>> f90nml.write(mynml, "mynml.nml")
>>> readnml = f90nml.read("mynml.nml")
>>> if readnml["section"]["falsevar1"]:
...    print("falsevar1 not False")
falsevar1 not False
>>> if readnml["section"]["falsevar2"]:
...    print("falsevar2 not False")
... 

```

It looks like it encloses ".true." and ".false." with quote marks in the written files, making it read them back in as strings instead of booleans.

Reply:
looks like f90nml is handling this, so what you have is correct :)

https://f90nml.readthedocs.io/en/latest/usage.html#f90nml.parser.Parser.strict_logical